cms/lib/xblock/tagging/tagging.py

```diff
@@ -43,7 +43,8 @@ def student_view_aside(self, block, context):  # pylint: disable=unused-argument
         Display the tag selector with specific categories and allowed values,
         depending on the context.
         """
-        if isinstance(block, CapaModule):
```

Comment:
@andy-armstrong @cpennington another variant how to check that the problem content is displayed in studio is to check that `runtime` class is an instance of `PreviewModuleSystem` class